cn_indian_payroll/cn_indian_payroll/overrides/tax_declaration.py

```
@@ -78,6 +79,84 @@ def update_value_in_form(self):
 
 
 
+#-----------validate section10 components on employee is eligible or not----------------
+
+    def validation_on_section10(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `validation_on_section10`

Add return type annotation: `None`

(ANN201)